lta/rest_server.py

```diff
@@ -15,10 +15,12 @@
 from motor.motor_tornado import MotorClient, MotorDatabase  # type: ignore
 import pymongo  # type: ignore
 from pymongo import MongoClient
-from rest_tools.client import json_decode  # type: ignore
+from rest_tools.utils.json_util import json_decode  # type: ignore
```

Comment:
I don't think you'll need to import this if you're using the new getters

Reply:
I did the `/Metadata` routes with the new getters.
I'd rather save a refactor of the existing routes (which need `json_decode`) for another PR.